src/client/components/OIbForm/OIbCheckBox.jsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { FormGroup, Label, Input, CustomInput } from 'reactstrap'
+import objectPath from 'object-path'
+import { ConfigContext } from '../../context/configContext.jsx'
 
-const OIbCheckBox = ({ label, value, name, onChange, defaultValue, switchButton }) => {
+const OIbCheckBox = ({ label, value, name, onChange, defaultValue, switchButton, dependency, dependencyValues }) => {
```

Comment:
I like the concept of dependencies but I guess it should not be specific to only check boxes. This adds some complexity to OIBCheckBox that should not be handled at this level. 

I tend to think that this should be implemented at the Form level for every Component and in a different PR. Actually, the option should allow to disable or hide the component.

Reply:
You're right, we can handle on Form level for all kind of components, do we want to change this on this PR or should we create a separate ticket and handle in a dedicated PR? Do we need to change anything related to this on this PR?

Reply:
I would prefer this to be separate PR. For now, we can remove the dependencies of OIBCheckBox so we can close this PR and reopen a new one.

Reply:
The dependency of OIBCheckBox was removed.